tox.ini

```diff
@@ -24,7 +24,14 @@ deps =
     sphinx32: Sphinx<3.3
     sphinx33: Sphinx<3.4
     sphinx34: Sphinx<3.5
+    sphinx35: Sphinx<3.6
+    sphinx40: Sphinx<4.1
+    sphinx41: Sphinx<4.2
+    sphinx42: Sphinx<4.3
+    sphinx43: Sphinx<4.4
+    sphinx44: Sphinx<4.5
     sphinxlatest: Sphinx
+    sphinxmaster: git+https://github.com/sphinx-doc/sphinx.git@master
```

Comment:
I'm not running sphinxmaster on circle since it could fail, but it's useful for testing future changes locally.